tests/sharding/test_tester_chain_handler.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from viper import compiler
+
+from eth_tester.backends.pyevm.main import get_default_account_keys
+
+from evm.utils.address import generate_contract_address
+
+import eth_utils
+
+from evm.chains.sharding.mainchain_handler.config import (
+    GASPRICE,
+    PASSPHRASE,
+    TX_GAS,
+)
+
+from evm.chains.sharding.mainchain_handler.vmc_utils import (
+    decode_contract_call_result,
+    mk_contract_tx_obj,
+)
+
+from evm.chains.sharding.mainchain_handler.backends.tester_chain_handler import (
+    TesterChainHandler,
+)
+
+keys = get_default_account_keys()
+
+@pytest.fixture(scope='function')
+def chain_handler():
+    return TesterChainHandler()
+
+def test_tester_chain_handler(chain_handler):
+    chain_handler.mine(1)
+    code = """
+num_test: public(num)
```

Comment:
For readability can this block of code get moved to a constant at the top level of the module so that the indentation isn't broken up here?

Reply:
no problem